config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_containernodepools.container.cnrm.cloud.google.com.yaml

```diff
@@ -271,6 +273,16 @@ spec:
                   enablePrivateNodes:
                     description: Whether nodes have internal IP addresses only.
                     type: boolean
+                  networkPerformanceConfig:
```

Comment:
Similarly, it's nice to add many new features to ContainerNodePool, but test coverage will be needed. Considering the size of the change, I think you might want to focus the taint feature.

Reply:
@maqiuyujoyce Thank you for your feedback. You are absolutely right. I will focus only on the `taint` feature. I will revert changes to the most of the `go` files I took from the terraform provider code and focus on `taints`.